tools/inference_engine/src/impl/nodes/convolution.cpp

```diff
@@ -37,4 +38,21 @@ namespace inference_engine
         return true;
     }
 
+    bool GpuConvolution::fuse_with(const GpuElementwiseAdd* elemwise)
+    {
+        assert(elemwise);
+        std::cout << "activation fuse with... elementwise_add\n";
+        outputs_ = elemwise->get_outputs();
+        for (auto& out : outputs_)
+            GpuNode::replace_input(out, elemwise, this);
+
+        // we have to update out inputs, activation contains only one input, bur for elementwise_add we have bring its second input
+        auto& elem_inputs = elemwise->get_inputs();
+        assert(elem_inputs.size() == 2 && (elem_inputs[0] == this || elem_inputs[1] == this));
+        inputs_.push_back(elem_inputs[0] == this ? elem_inputs[1] : elem_inputs[0]);
```

Comment:
Shouldn't second's node have added new output (this convolution node)? Or maybe this logic is already here and I'm missing it?